logic/standalone/sort_conflict.py

```diff
@@ -0,0 +1,190 @@
+    pruned_dict = conflict.PruneConflicts(playdo, conflict_dictionary)
+    conflict.FixConflicts(playdo, pruned_dict)
+'''
+
+import logic.common.log_utils as log
+import logic.common.tiled_utils as tiled_utils
+
+#--------------------------------------------------#
+'''Variables'''
+
+# TODO delete the whole section since there is no variable/constants here
+
+
+
+
+#--------------------------------------------------#
+'''Public Functions'''
+
+def OrganizeObjectsBySortVal(playdo, list_scan_obj):
+	'''Given a list of all existing Tiled objects in a level, will return a dictionary:
+		KEY   - the unique sort values within the level (string)
+		VALUE - a list of Tiled objects that share the same sort value
+	'''
+	log.Info("\n--- Checking for sort value conflicts ---\n")
+
+	# Create the dictionary with each unique sort value as keys
+	sortval_to_objects_map = {}
+	for scanned_obj in list_scan_obj:
+		curr_obj_list = playdo.GetAllObjectsWithName(scanned_obj)
+		log.Info( f'# of \"{scanned_obj}\": {len(curr_obj_list)}' )
+		for obj in curr_obj_list:
+			curr_string = tiled_utils.GetPropertyFromObject(obj, '_sort')
+			if curr_string == None: continue  # When object does not contain `_sort`
+			if curr_string == '': continue    # When property has property but no value
+			curr_string = curr_string.split('.')[0]	# For checking the "fixed" sort
+			if not curr_string in sortval_to_objects_map:
+				sortval_to_objects_map[curr_string] = []
+			sortval_to_objects_map[curr_string].append(obj)
+
+	log.Must("\n--- Finished checking conflicts! ---\n")
+	return sortval_to_objects_map
+
+
+
+def PrintPotentialConflicts(playdo, sortval_to_objects_map, list_scan_obj):
+	'''Print the dictionary that has been checked and sorted by their sort values'''
+
+	# Find the max length in object names and layer names for indentation
+	max_obj_name_len = len( max(list_scan_obj, key=len) )
+	max_layer_name_len = 1
+	list_layer_name = []
+	for obj_layer in playdo.level_root.findall('.//objectgroup'):
+		name_len = len( tiled_utils.GetNameFromObject(obj_layer) )
+		if max_layer_name_len < name_len: max_layer_name_len = name_len
+
+	# Set the parent of all objects
+	parent_map = {child: parent for parent in playdo.level_root.iter() for child in parent}
+
+	# Print for each sort value
+	for key_sort, list_obj in sortval_to_objects_map.items():
+		log.Extra( '\n--------------------------------------------------' )
+		log.Info( f"{key_sort} has {len(list_obj)} elements" )
+		for obj in list_obj:
+			log.Extra( f'  {PrintObjInfo(obj, GetParentName(obj, parent_map), max_obj_name_len, max_layer_name_len)}' )
+	log.Extra('\n--------------------------------------------------')
+	log.Must('\n--- Finished printing conflicts! ---\n')
+
+
+
+
+
+
+
+#--------------------------------------------------#
+'''Utility'''
+
+def _GetSort( tiled_object ):
+    return tiled_utils.GetPropertyFromObject(tiled_object, '_sort')
+
+def GetParentName(obj, parent_map):
```

Comment:
I don't know how to name it either! It's just a map that I copied somewhere I can't recall.

I tried adding more to the description instead, LMK if that still confuses you by the time you see it.

Also it often doesn't matter whether something is treated as an object, objectgroup, XML tree object, or whatever applicable. They're probably handled in the same manner anyway, so there's little reason to distinguish them since it might over-complicate things IMO.